modules/bibupload/lib/bibupload.py

```diff
@@ -288,6 +289,7 @@ def bibupload(record, opt_mode=None, opt_notimechange=0, oai_rec_id="", pretend=
                 error = None
         else:
             write_message("   Note: 005 already existing upon inserting of new record. Keeping it.", verbose=2)
+        original_005 = time.strftime("%Y-%m-%d %H:%M:%S", time.strptime(record['005'][0][3].split('.')[0], "%Y%m%d%H%M%S"))
```

Comment:
In this case we should rename the variable, because `original_005` does not represent "original 005", as it were.  (See also the above comment that it could be that `original_005=None`, if somebody only reads the `update_bibrec_date()` function signature.)  
Maybe call it `initial_creation_date` or something?